### PR TITLE
[noticket] Updated walkthrough because we're GAE now, not cloud functions

### PR DESCRIPTION
**Pull request by helgridly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +40,4 @@ You should make mypy happy before opening a PR. Note that errors in some modules
 
 ### Deployment
 
-Run `./deployall.sh`. You will need a copy of `secrets.yaml` in the root of the repo, which is not checked in because it's full of secrets. `secrets.conf` is the to-be-templated version, though the secrets aren't stored in vault and there's no templating mechanism built yet. Ask Hussein if you want the secrets.
-
-To smoke test a deployment, run `./smoketest.sh`. This will do a few things and prompts you to visually check the output to see if it looks right.
\ No newline at end of file
+Run `./deployall.sh`. You will need a copy of `app.yaml` in the root of the repo, which is not checked in because it's full of secrets. `app.yaml.ctmpl` is the to-be-templated version, though the secrets aren't stored in vault and there's no templating mechanism built yet. Ask Hussein if you want the secrets.
```
Comment (MatthewBemis) [inline README.md:43]: is there a ticket to store this stuff in vault?

**Review by MatthewBemis — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (davidangb) [reply README.md:43]: and update this walkthrough when we are vault-enabled!

File: WALKTHROUGH.md [hunk 2]
```diff
@@ -1,103 +1,117 @@
...
+
+Once the translation is complete it sends a pubsub message to Rawls to import the batchUpsert file.
+
+#### Status handling `/app/status.py`
 
-This is a placeholder background function. When it grows up it wants to be the "chunk" import task, but for now it pulls a parameter out of the Pub/Sub message and looks it up in the database. Because background functions don't return anything, it just logs the result, which you can see in the Cloud Function logs in the GCP console.
+This handles getting the status for existing imports. It also provides a pubsub message that "trusted external services" (i.e. Rawls, which is the only other SA allowed to write to the import service's pubsub queue) can send to update the status of an import.
```
Comment (davidangb) [inline WALKTHROUGH.md]: nit: "It also provides a pubsub message" should  be something like "it also accepts a pubsub message" or "it also provides a pubsub message listener" or "it defines and responds to a pubsub message"

**Review by davidangb — APPROVED** [review]
:+1: one nit inline

File: README.md (hunk 1 above)
Comment (helgridly) [reply README.md:43]: It's wrapped up in the "get it on real envs" ticket.

File: WALKTHROUGH.md (hunk 2 above)
Comment (helgridly) [reply WALKTHROUGH.md]: You're right, this wording is terrible

**Pull request merged**